src/main/java/seedu/address/ui/ProjectDisplayPanel.java

```diff
@@ -159,6 +177,10 @@ public void showTodosTab() {
         tabPane.getSelectionModel().select(TODOS_TAB);
     }
 
+    public void setMainWindow(MainWindow mainWindow) {
+        this.mainWindow = mainWindow;
```

Comment:
Use setter instead of constructor here so that display panel can be tested in isolation (without being dependent on main window.